src/addrdb.cpp

```diff
@@ -49,7 +49,10 @@ bool SerializeFileDB(const std::string& prefix, const fs::path& path, const Data
         return error("%s: Failed to open file %s", __func__, pathTmp.string());
 
     // Serialize
-    if (!SerializeDB(fileout, data)) return false;
+    if (!SerializeDB(fileout, data)) {
+        fileout.fclose();
```

Comment:
This is an autofile - doesn't it close automatically on release?

Reply:
Maybe remove explicit close and move `fileout` to an inner scope?